Sources/Plasma/FeatureLib/pfDXPipeline/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+set(pfDXPipeline_SOURCES
+    hsGDirect3D.cpp
+    plDXDevice.cpp
+    plDXDeviceRefs.cpp
+    plDXEnumerate.cpp
+    plDXPipeline.cpp
+    plDXPixelShader.cpp
+    plDXShader.cpp
+    plDXTextFont.cpp
+    plDXVertexShader.cpp
+)
+
+set(pfDXPipeline_HEADERS
+    hsGDirect3D.h
+    plDXBufferRefs.h
+    plDXDevice.h
+    plDXDeviceRef.h
+    plDXEnumerate.h
+    plDXLightRef.h
+    plDXPipeline.h
+    plDXPixelShader.h
+    plDXRenderTargetRef.h
+    plDXSettings.h
+    plDXShader.h
+    plDXTextFont.h
+    plDXTextureRef.h
+    plDXVertexShader.h
+)
+
+plasma_library(pfDXPipeline
+    SOURCES ${pfDXPipeline_SOURCES} ${pfDXPipeline_HEADERS}
+    PRECOMPILED_HEADERS Pch.h
+)
+target_link_libraries(pfDXPipeline
+    PUBLIC
+        CoreLib
+        pnKeyedObject
+        pnModifier
+        pnNucleusInc
+        plGImage
+        plProgressMgr
+        plScene
+        plStatusLog
+    PRIVATE
+        pnMessage
+        pnNetCommon
+        pnSceneObject
+        plAudio
+        plAvatar
+        plClientResMgr
+        plDrawable
+        plGLight
+        plIntersect
+        plMessage
+        plNetClient
+        plResMgr
+        plSurface
+        pfCamera # plCaptureRender, plDynamicEnvMap, plDXPipeline
+        pfGameGUIMgr # plCaptureRender
+        ${DirectX_LIBRARIES}
+    INTERFACE
+        pnFactory
```

Comment:
Have all of these been validated to be necessary? Spurious dependencies are a killer for build times with MSBuild, and I don't think `plCaptureRender` or `plDynamicEnvMap` are likely to be in `pfDXPipeline`.

Reply:
These have not been validated